test/integration/record_test.rb

```diff
@@ -61,5 +61,19 @@ class RecordTest < ActionDispatch::IntegrationTest
         assert(value.text.include?('Economic analysis for business decisions'))
       end
     end
+
+  test 'view online button is shown' do
+    VCR.use_cassette('record: article', allow_playback_repeats: true) do
+      get record_url, params: { db_source: 'aci', an: '123877356' }
+      assert_select 'a.button-primary', text: 'View online'
+    end
+  end
+
+  test 'check for online copy link shown when fulltext URL is SFX' do
```

Comment:
The EDS API provides a mechanism to know whether it is likely full text or not. That may not have made it to the Gem. If it did not, we probably want to open a feature request as many SFX links are known to be full text. You can't tell by the URL pattern, but EDS knows due to 🔮 (or more likely it knows because we told them our holdings).